js/components/ChartForm.vue

```diff
@@ -179,6 +187,8 @@ import {
   fieldLabel
 } from '@/util';
 
+const userDateFormat = 'MM/DD/YYYY';
```

Comment:
I don't think we need to deal with this now, but REDCap allows users to dictate their preferred date format. I'm not sure if this is available to us via the SDK, but it might be useful at some point.

Reply:
This is a good point. Right now I'm just trying to mimic the behavior of the jQuery version, so I opened issue #45 to track this.